e/easyeffects/stone.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# SPDX-FileCopyrightText: © 2020-2025 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : easyeffects
```

Comment:
I took a look at some other distros packages and I think we want lsp-plugins packaged as well. (and added as a rundep). Specifically the lv2 plugins that it builds.

Reply:
I've been trying to get lsp-plugins to build sadly haven't had much success I'll post the stone below maybe you can solve it
https://bpa.st/DQMQ